.github/workflows/publish-on-tag.yml

```diff
@@ -37,3 +37,15 @@ jobs:
           npm config set registry 'https://wombat-dressing-room.appspot.com/'
           npm config set '//wombat-dressing-room.appspot.com/:_authToken' '${NPM_TOKEN}'
           npm publish
+
+  post-publish:
+    needs: publish
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Bump to Post Version
+        run: |
+          VERSION=$(node utils/apply_post_version.js)
+          npm run doc
+          git commit -m 'chore: bump version to v$VERSION'"
```

Comment:
I think GitHub Actions don't have write access to the repo?

Reply:
They do. We might wanna configure the user name + email though. In case it’s useful, I used this for my old COVID-19 vaccination project: https://github.com/mathiasbynens/covid-19-vaccinations-germany/blob/4d8f58d7da0a3aedb3cb463454559d8745f2ede8/.github/workflows/update.yml#L34-L48

Reply:
what about `permissions: read-all` at the top of the file? I thought it restricts the access? (although could not find a good doc about what permissions are meant by that)

Reply:
I'll remove the `read-all` permission. As for the username and email, I guess we can just use some trivial one.

Reply:
Done.